doc/en/reference.rst

```diff
@@ -1192,6 +1192,45 @@ passed multiple times. The expected format is ``name=value``. For example::
         [pytest]
         junit_suite_name = my_suite
 
+.. confval:: log_auto_indent
+
+    Allow selective auto-indentation of multiline log messages.
+
+    Supports command line option ``--log-auto-indent [value]``
+    and config option ``log_auto_indent = [value]`` to set the
+    auto-indentation behavior for all logging.
+
+    ``[value]`` can be:
+        * True or On - Dynamically auto-indent multiline log messages
+        * False or Off or 0 - Do not auto-indent multiline log messages
+        * [positive integer] - auto-indent multiline log messages by [value] spaces
```

Comment:
Please mention that auto-indentation is turned off by default.

Reply:
done